src/CLI/Console.php

```diff
@@ -133,20 +133,24 @@ static public function exit(int $status = 0): void
      */
     static public function execute(string $cmd, string $stdin, string &$stdout, string &$stderr, int $timeout = -1): int
     {
+        $cmd = '( ' . $cmd . ' ) 3>/dev/null ; echo $? >&3';
```

Comment:
Is there any chance a command with ' inside of it can cause this function to fail? If so could we add escape characters?

Reply:
`echo 'Hello World'` worked fine. I think `'`symbols are only part of PHP syntax in this case.

Reply:
Alright, lgtm